index.d.ts

```diff
@@ -1,3 +1,83 @@
+import {CamelCase, PascalCase} from 'type-fest';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type EmptyTuple = [];
+
+/**
+ * Return a default type if input type is nil.
+ *
+ * @template T input type
+ * @template U default type
+ */
+type WithDefault<T, U extends T> = T extends undefined | void | null ? U : T;
+
+/**
+ * Check if an element is included in a tuple.
+ *
+ * TODO: Remove this once https://github.com/sindresorhus/type-fest/pull/217 is merged.
+ */
+type IsInclude<List extends readonly unknown[], Target> = List extends undefined
+	? false
+	: List extends Readonly<EmptyTuple>
+		? false
+		: List extends readonly [infer First, ...infer Rest]
+			? First extends Target
+				? true
+				: IsInclude<Rest, Target>
+			: boolean;
+
+/**
+ * Append a segment to dot-notation path.
+ */
+type AppendPath<S extends string, Last extends string> = S extends ''
+	? Last
+	: `${S}.${Last}`;
+
+/**
+ * Convert keys of an object to camelcase strings.
```

Comment:
Don't prefix with `*`
```suggestion
Convert keys of an object to camelcase strings.
```